lib/command.mjs

```diff
@@ -0,0 +1,18 @@
+import common from './common';
+import program from './program';
+import doctest from '..';
+
+Promise.all (program.args.map (function(path) {
+  return doctest (path, program).then (function(results) {
+    console.log (path, results);
+    return results.reduce (function(status, tuple) {
+      return tuple[0] ? status : 1;
+    }, 0);
+  });
+})).then (function(statuses) {
+  process.exit (Math.max.apply (null, statuses));
+}, function(err) {
+  console.error (err.stack); //TMP
+  process.stderr.write (common.formatErrors ([err.message]));
+  process.exit (1);
+});
```

Comment:
This code is exactly the same as in the cjs version of command. Maybe move it to a function in `common.js`?